README.md

```diff
@@ -1,7 +1,7 @@
 Alluxio (formerly Tachyon)
 =======
 
-The master branch is in version 1.8.0-SNAPSHOT:
+The master branch is in version 1.9.0-SNAPSHOT:
```

Comment:
Should we also update the stable release versions below?

Reply:
Not until 1.8.0 is out the door